modules/manage/partials/monitor-redpanda.adoc

```diff
@@ -281,11 +281,15 @@ To detect unexpected idling, you can query the rate of change as a percentage of
 rate(redpanda_cpu_busy_seconds_total[5m])
 ----
 
-[NOTE]
+[TIP]
 ====
-While CPU utilization at the host-level can typically appear high (for example, 99-100% utilization) when I/O events like message arrival occur, the actual Redpanda process utilization is likely low and can be monitored with `redpanda_cpu_busy_seconds_total`.
+While CPU utilization at the host-level can typically appear high (for example, 99-100% utilization) when I/O events like message arrival occur, the actual Redpanda process utilization is likely low, in contrast to what system-level metrics might indicate such as those provided by the `top` command. 
+
+This high host-level CPU utilization happens because Redpanda uses Seastar, which runs event loops on every core (also referred to as a _reactor_), constantly polling for the next task. This process never blocks and will increment clock ticks. It doesn't necessarily mean that Redpanda is busy.
+
+Use `redpanda_cpu_busy_seconds_total` (public metrics) or xref:reference:internal-metrics-reference.adoc#vectorized_reactor_utilization[`vectorized_reactor_utilization`] (internal metrics) to monitor the actual Redpanda CPU utilization. When these particular metrics approach 100% utilization, you will then also start to see higher produce and consume <<latency,latency>>.
 
-This high host-level CPU utilization happens because Redpanda uses Seastar and constantly polls and increments clock ticks, and it doesn't necessarily mean that Redpanda is busy.
+If achieving very low latency consume is not a priority, consider tuning the xref:reference:tunable-properties.adoc#fetch_reads_debounce_timeout[`fetch_reads_debounce_timeout`] property. The default setting at 1ms means that Redpanda pings the partition once per millisecond of the given client poll cycle. Setting the duration at a higher value, for example up to 10ms, can help reduce CPU utilization.
```

Comment:
Might also add something like:

Increasing above 10ms is not recommended as this will provide diminishing returns at the expense of further increasing average message consume latency.

Reply:
btw, looks good otherwise.